modules/ROOT/pages/metadata-input.adoc

```diff
@@ -12,6 +12,8 @@ while Sources, Configurations and Connections will always have static metadata f
 
 == Declaring an InputTypeResolver
 
+The `InputTypeResolver` assigns a `MetadataType` to a class to resolve the input of the component.
```

Comment:
This is still incorrect and I think that in the section above we already describe what a InputTypeResolver does. I'd scrap this.